.github/workflows/release_dev.yml

```
@@ -91,10 +95,26 @@ jobs:
+      - name: Debug tag and commit 2
+        run: |
+          echo "Tag name: ${{ steps.tag_info.outputs.tag_name }}"
+          echo "Target commitish: ${{ github.ref_name }}"
+          git rev-parse ${{ github.ref_name }} || echo "Commit not resolved"
+
+  gamemaker_build:
+    name: Build
+    uses: ./.github/workflows/gamemaker_build.yml
+    secrets: inherit
+    needs: prepare_release
+    if: needs.build_needed.outputs.needed == 'true'
```

actionlint (1.7.4)

property "build_needed" is not defined in object type {prepare_release: {outputs: {build_date: string; tag_name: string}; result: string}}

(expression)